server/index.js

```diff
@@ -1,9 +1,45 @@
 const express = require('express');
 const db = require('../database/database');
+const sessionData = require('../data-simulation/sessionData');
 
 const app = express();
 
-app.get('/', (req, res) => res.send('Tetraflix User Profiles Service'));
+app.get('/', (req, res) => {
+  res.status(200).send('Tetraflix by Tetragon - User Profiles Service');
+});
+
+// GET request to get total movie_history row count
+// generates sessions data then write into db
+app.get('/movieHistory', (req, res) => {
+  const sessions = sessionData.simulateData();
+  const start = new Date();
+  return Promise.all(sessions.map((session) => {
+    const { userId } = session;
```

Comment:
Is there a reason you used const { userId } instead of const userId? I'm a little unclear on this.

Reply:
I used const { userId } to assign session.userId to userId, not session to userId.